legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -572,7 +572,7 @@ def party_role_revision_json(filing, party_role_revision, is_ia_or_after) -> dic
     def get_party_revision(filing, party_id) -> dict:
         """Consolidates all party changes up to the given transaction id."""
         business = BusinessService.fetch_business_by_filing(filing)
-        if filing.legal_entity_id:
+        if filing.legal_entity_id or business.legal_entity_id:
```

Comment:
i don't think this change is required?

Reply:
FM1064603, filing_id: 148571.

In this case, the filing doesn't have the legal_entity. it has the alternate_name_entity.
And alternate_name_entity has the legal_entity.

I think you mentioned that filing pull back the legal_entity in this case

Reply:
Yes, I mentioned that.  But I think the logic needs to be more precise.  Anyways, we can pick this up in the new ticket.